content/Rust-1.88.0.md

```diff
@@ -0,0 +1,109 @@
++++
+path = "2025/06/26/Rust-1.88.0"
+title = "Announcing Rust 1.88.0"
+authors = ["The Rust Release Team"]
+aliases = ["releases/1.88.0"]
+
+[extra]
+release = true
++++
+
+The Rust team is happy to announce a new version of Rust, 1.88.0. Rust is a programming language empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via `rustup`, you can get 1.88.0 with:
+
+```
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`](https://www.rust-lang.org/install.html) from the appropriate page on our website, and check out the [detailed release notes for 1.88.0](https://doc.rust-lang.org/stable/releases.html#version-1880-2025-06-26).
+
+If you'd like to help us out by testing future releases, you might consider updating locally to use the beta channel (`rustup default beta`) or the nightly channel (`rustup default nightly`). Please [report](https://github.com/rust-lang/rust/issues/new/choose) any bugs you might come across!
+
+## What's in 1.88.0 stable
+
+### Cargo automatic cache cleaning
```

Comment:
Just a suggestion, but I might put this last, since it is maybe the least exciting thing here. 😆  I suspect people are most excited about let-chains by far.

(Personally I'm most excited for boolean configurations, but I do a lot of weird testing.)

Reply:
Yea, I think for most people that target directories dominate significantly.